File: datasets/IR-Plag/convertai-original-java-rest-python/case-02/plagiarized/L5/08/Kasus2L5.py
import math

def main():
    # Enter radius of the cylinder
    print("Enter the radius and length of a cylinder: ", end="")
    radius = float(input())
    # Calculate Area
    area = calculate_area(radius)
    # Get length
    length = float(input())
    # Print area cylinder
    print("The area is", area)
    # Calculate Volume
    volume = calculate_volume(area, length)
    # Print volume cylinder
    print("The volume of the cylinder is", volume)

def calculate_area(radius):
    return radius * radius * math.pi

def calculate_volume(area, length):
    return area * length

if __name__ == "__main__":
    main()

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-02/plagiarized/L5/08/Kasus2L5.ts
import { Scanner } from 'readline-sync';

/**
 *
 * @author B15130F5DDB6B5F1622EF91DAC4C1AAE
 */
function main(): void {
    // Enter radius of the cylinder
    console.log("Enter the radius and length of a cylinder: ");
    const in: Scanner = new Scanner();
    //input radius
    const radius: number = in.nextFloat();
    // Hitung Area
    const area: number = calculateArea(radius);
    //input Length
    const length: number = in.nextFloat();
    // Print area cylinder
    console.log("The area is " + area);
    //Hitung Volume
    const volume: number = calculateVolume(area, length);
    // Print volume cylinder
    console.log("The volume of the cylinder is " + volume);
}

function calculateArea(radius: number): number {
    return radius * radius * 3.14159;
}

function calculateVolume(area: number, length: number): number {
    return area * length;
}

main();

